CHANGELOG.md

```diff
@@ -71,6 +71,8 @@
 
 * Add GitHub Actions Logging reporter (`github-actions-logging`).  
   [Norio Nomura](https://github.com/norio-nomura)
+* Make `control_statement` rule substitution correctable.
+  [MaxHaertwig](https://github.com/maxhaertwig)
```

Comment:
```
* Make `control_statement` rule substitution correctable.  
  [MaxHaertwig](https://github.com/maxhaertwig)
```
It seems that you missed two spaces after the first line.

Reply:
Thanks for noticing